### DISCUSSION TITLE
Transferred Ownership

### DISCUSSION THREAD

**Discussion by benjitrosch** [daisyui/react-daisyui#8] [General]
Hey @saadeghi , this new repo was transferred over from my account so we could keep the stars we previously had. However, in transferring ownership, it seems @christianblandford and myself lost all permissions on the repo 😅 

Any chance you could add us back as admins of the page like before? That way we can set up the Github Pages for our v1.0.0 release. Thanks!

**Comment by saadeghi** [comment]
Done ✅

**Comment by saadeghi** [comment]
I think I can also add a subdomain like `react.daisyui.com`. Is that okay?

**Reply by benjitrosch** [reply]
That would be amazing!

**Reply by saadeghi** [reply]
https://react.daisyui.com/ ✅
